lib/libs/email/content/withdrawPackage/index.tsx

```diff
@@ -48,4 +50,53 @@ export const withdrawPackage: AuthoritiesWithUserTypesTemplate = {
       };
     },
   },
+  [Authority["1915b"]]: {
+    cms: async (
+      variables:
+        | (Events["CapitatedInitial"] & CommonEmailVariables & { emails: EmailAddresses })
+        | (Events["ContractingInitial"] & CommonEmailVariables & { emails: EmailAddresses }),
+    ) => {
+      return {
+        to: variables.emails.osgEmail,
+        subject: `Waiver Package ${variables.id} Withdraw Request`,
+        body: await render(<WaiverCMSEmail variables={variables} />),
+      };
+    },
+    state: async (
+      variables:
+        | (Events["CapitatedInitial"] & CommonEmailVariables & { emails: EmailAddresses })
+        | (Events["ContractingInitial"] & CommonEmailVariables & { emails: EmailAddresses }),
+    ) => {
+      return {
+        to: [`${variables.submitterName} <${variables.submitterEmail}>`], // TODO: change to ALL state users
```

Comment:
ICYMI – `// TODO` still relevant?

Reply:
unfortunately it is 😢 
getting all the state users is a part of this ticket: https://jiraent.cms.gov/browse/OY2-31140

Reply:
Sounds good. Resolving!